worlds/peaks_of_yore/__init__.py

```diff
@@ -0,0 +1,145 @@
+import logging
+from typing import Any
+
+from Options import Toggle, OptionError
+from worlds.AutoWorld import World, WebWorld
+from BaseClasses import Tutorial, Item
+from .options import PeaksOfYoreOptions, Goal, StartingBook, RopeUnlockMode, StartingHands, poy_option_groups, \
+    poy_option_presets
+from .data import *
+
+from .regions import create_poy_regions, RegionLocationInfo
+
+
+class PeaksOfYoreItem(Item):
+    game = "Peaks of Yore"
+
+
+class PeaksOfWeb(WebWorld):
+    rich_text_options_doc = True
+    theme = "stone"
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Peaks of Yore Mod.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["c0der23"]
+    )]
+    option_groups = poy_option_groups
+    options_presets = poy_option_presets
+
+
+class PeaksOfWorld(World):
+    """
+    Peaks of Yore is a first-person physics-based "climb-em-up" adventure set in 1887.
+    Steel your nerves and perfect your climbing skills as you ascend the rock wall, traverse difficult routes,
+    and encounter many challenges and obstacles.
+    """
+    game = "Peaks of Yore"
+    options_dataclass = PeaksOfYoreOptions
+    options: PeaksOfYoreOptions
+    web = PeaksOfWeb()
+    item_name_to_id = item_name_to_id
+    location_name_to_id = all_locations_to_ids
+    topology_present = True
```

Comment:
Is topology_present really necessary for this world? It outputs to the spoiler log the entrance layout, but this world doesn't have any entrance rando, right?